.github/workflows/build.yml

```diff
@@ -57,6 +57,7 @@ jobs:
 
     - name: Install
       run: |
+        pip --verbose list
```

Comment:
Is this necessary?

Reply:
Extra debug output isn't necessary, until things do not work, and you do not know why :-)

Not strictly necessary for the build to work.  Let me know if you want me to remove it, and I will.

Reply:
Removed in commit 5.